package.json

```diff
@@ -11,31 +11,29 @@
   "bugs": {
     "url": "https://github.com/avoidwork/tiny-lru/issues"
   },
+  "files": [
+    "lib",
+    "*.d.ts"
+  ],
   "license": "BSD-3-Clause",
-  "main": "lib/tiny-lru.js",
-  "module": "lib/tiny-lru.es6.js",
-  "types": "lib/tiny-lru.d.ts",
+  "source": "src/lru.js",
+  "browser": "lib/tiny-lru.js",
+  "main": "lib/tiny-lru.cjs.js",
+  "module": "lib/tiny-lru.esm.js",
+  "types": "tiny-lru.d.ts",
```

Comment:
so essentially

- we want nodejs code `main` property to read `lib/tiny-lru.cjs.js` which was `lib/tiny-lru.js` before.

- we are changing `types` to just point at file at root and avoid copy to lib.

- we are also renaming `module` entry point from `lib/tiny-lru.es6.js` to `lib/tiny-lru.esm.js` -- This is to avoid versioning confusion in future. Releasing es2020 code for example won't need to change this file again in future.

- we are introducing a new `browser` property to publish our modules as `umd` -- read here about [browser property](https://docs.npmjs.com/files/package.json#browser)

- we are also removing un-minified version (we minify by default). I am planning to release a new version of `tslib-cli` which will start producing sourcemaps too, which is not available for now.